libs/docs/src/docs/no-version/what-is-a-devfile.md

```diff
@@ -3,16 +3,14 @@ title: What is a devfile
 description: What is a devfile
 ---
 
-You can use devfiles to automate and simplify your development process
-by adopting the existing devfiles that are available in the [public community registry](https://registry.devfile.io/viewer)
-or by authoring your own devfiles to record custom instructions to 
-configure and run your build environment as a YAML-formatted text file.
-You can make these devfiles available in the supporting build tools and 
-IDEs that can automatically process the devfile instructions to configure 
-and build a running application from a development project.
-
-Using the recommended best practices from the devfile, the tools and IDE
-can:
+`devfile.yaml` describes how to configure and run build environment for
+your project in a development container. You can create devfile from scratch or
+adopt existing from [public community registry](https://registry.devfile.io/viewer).
```

Comment:
```suggestion
The devfile, `devfile.yaml`, describes how to configure and run your workspace under a Kubernetes environment using a development container. You can create devfile workspace from scratch or adopt existing boilerplate devfile stack from the [public community registry](https://registry.devfile.io/viewer).
```

Reply:
The line is too long to see the diff.

Reply:
@michael-valdron the formatting is better, but the definition of "workspace" is vague. The descriptions fits the example on how to run VSCode session in Kubernetes cluster, which is not the case. It is not what is the difference between "devfile" and "devfile workspace" here..

Also it looks like it is impossible to use devfile without Kubernetes. Not sure if that's the case.